app/src/main/java/org/p2p/wallet/infrastructure/network/interceptor/RpcInterceptor.kt

```diff
@@ -30,6 +45,51 @@ class ServerErrorInterceptor(
         }
     }
 
+    private fun createRpcRequest(chain: Interceptor.Chain): Request {
+        val request = chain.request()
+
+        val json = getRequestJson(request)
+
+        val url = request.url
+
+        val environmentUrl = if (json.getString("method") == "getConfirmedTransaction") {
```

Comment:
would be nice to make this condition as function or variable.
And let's move method name to constants